apps/loan/src/components/PageLlamaMarkets/cells/LineGraphCell.tsx

```diff
@@ -0,0 +1,58 @@
+import { LendingSnapshot, useLendingSnapshots } from '@/entities/lending'
+import { LendingVault } from '@/entities/vaults'
+import { Line, LineChart } from 'recharts'
+import { useTheme } from '@mui/material/styles'
+import { DesignSystem } from '@ui-kit/themes/design'
+import Box from '@mui/material/Box'
+import Skeleton from '@mui/material/Skeleton'
+
+const graphSize = { width: 172, height: 48 }
```

Comment:
Would be nice if this could be made responsive and / or determined by the table that uses this cell instead of having the size hardcoded in the cell itself.

Reply:
Yeah good point. We should discuss with tutti